test/tests/misc/signal_handler/simple_transient_diffusion.i

```diff
@@ -0,0 +1,53 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  uniform_refine = 3
```

Comment:
This is no longer simple transient diffusion. Let's call it "scaled simple transient diffusion" or something along those lines to convey that it's not just a copy.